src/views/hub/index.php

```diff
@@ -102,6 +102,12 @@
                         'linkOptions' => ['data-action' => 'set-rack-no'],
                         'visible' => Yii::$app->user->can('hub.update'),
                     ],
+                    Yii::$app->user->can('server.update') ? [
```

Comment:
Probably, `hub.update`?

Reply:
Users with the `admin` role have this permission and can update unit quantity. Is it expected?